langDir/fr.json

```
@@ -86,15 +86,15 @@
       "1": {
         "m": "@.md:faq.a.1.l1 Vous pouvez en obtenir par le biais de notre @.md:faq.a.1.l2 fonction ou @.md:faq.a.1.l3",
         "l1": "[Kusama, $KSM.](@.common:links.kusama_network.href)",
-        "l2": "[Credit](@.common:links.rmrk_credit.href)",
+        "l2": "[Crédit](@.common:links.rmrk_credit.href)",
         "l3": "[par échange de garde](@.common:links.binance_exchange.href)."
       },
       "2": {
-        "m": "Nous vous recommandons d'utiliser l'@.md:faq.a.2.l1 pour l'instant. Plus tard, nous prévoyons d'ajouter une expérience mobile d'abord. Dans tous les cas, <b> sauvegardez votre graine, c'est votre droit de propriété et le type de valeur.</b>",
+        "m": "Nous vous recommandons d'utiliser l'@.md:faq.a.2.l1 pour l'instant. Plus tard, nous prévoyons d'ajouter une expérience orientée \"mobile-first\". Dans tous les cas, <b> sauvegardez votre graine, c'est votre droit de propriété et le type de valeur.</b>",
         "l1": "[extension Polkadot](@.common:links.polkadot_js_extension.href)"
       },
       "3": {
-        "m": "Cliquez sur @.md:faq.a.3.l1 Vous devez d'abord monnayer votre collection, puis le jeton. Si vous voulez filmer cela, envoyez-nous un DM!",
+        "m": "Cliquez sur @.md:faq.a.3.l1 Vous devez d'abord monnayer votre collection, puis le jeton. Si vous voulez filmer cela, envoyez-nous un MD!",
```

```suggestion
        "m": "Cliquez sur @.md:faq.a.3.l1 Vous devez d'abord monnayer votre collection, puis le jeton. Si vous voulez filmer cela, envoyez-nous un DM!",
```
DM would be better here

---

That's up to you guys. In French, we tend to use MD instead of the DM english version, but both are commonly understood.

---

> That's up to you guys. In French, we tend to use MD instead of the DM english version, but both are commonly understood.

I don't know which part of France, but MD means something else to me. "DM" has been in common language for some time, otherwise you can say "MP"

---

Indeed MP is better